pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -1165,15 +1025,20 @@ func TestChangefeedMissingDatabaseErr(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	cdcTest(t, func(t *testing.T, s TestServer, f cdctest.TestFeedFactory) {
-		expectErrCreatingFeed(t, f, `CREATE CHANGEFEED FOR DATABASE foo`, `database "foo" does not exist`)
+		expectErrCreatingFeed(t, f,
+			`CREATE CHANGEFEED FOR DATABASE foo`,
+			`database-level changefeed is not implemented yet`,
+		)
 	})
 }
 
 func TestChangefeedCannotTargetSystemDatabaseErr(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	cdcTest(t, func(t *testing.T, s TestServer, f cdctest.TestFeedFactory) {
-		expectErrCreatingFeed(t, f, `CREATE CHANGEFEED FOR DATABASE system`, `changefeed cannot target the system database`)
+		expectErrCreatingFeed(t, f,
+			`CREATE CHANGEFEED FOR DATABASE d`,
```

Comment:
nit: in keeping with the spirit of the original test, the changefeed should target the system database
```suggestion
			`CREATE CHANGEFEED FOR DATABASE system`,
```